locations/spiders/PureGym.py

```diff
@@ -0,0 +1,51 @@
+import json
```

Comment:
Can you rename the file so it's lower-cased? `puregym.py` probably

Reply:
Done